.mocharc.js

```diff
@@ -1,12 +1,15 @@
 // We can't import the `.mocharc.js` of the monorepo, otherwise we trigger its `setupBabel`.
+
 module.exports = {
-  extension: ['js', 'ts', 'tsx'],
+  extension: ['js', 'ts', 'tsx', 'jsx'],
```

Comment:
Not allowed, historical decision https://github.com/airbnb/javascript/pull/985#issuecomment-239145468 

```suggestion
  extension: ['js', 'ts', 'tsx'],
```